CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Change Log
 
+## 0.0.14 (Jully 2nd, 2020)
+
+- enhancement - Using vulnerability data from Snyk.
+- enhancement - Includes latest updated security vulnerabilities of `maven`, `node` and `pypi` ecosystems, also includes Snyk curated Security Advisories/Vulnerability Unique to Snyk.
```

Comment:
enhancement - Includes latest updated security vulnerabilities for `maven`, `node` and `pypi` ecosystems. It includes Snyk curated Security Advisories/Vulnerabilities, which are not available publicly.